backend/tests/services/test_local_device_command_service.py

```
@@ -0,0 +1,364 @@
+    monkeypatch.setattr(
+        command_service.device_service,
+        "get_device_online_info",
+        AsyncMock(return_value={"socket_id": "socket-123"}),
+    )
+    monkeypatch.setattr(command_service, "get_sio", lambda: mock_sio)
+
+    result = await command_service.local_device_command_service.execute_command(
+        user_id=7,
+        device_id="device-abc",
+        command="pwd",
+        path="/tmp",
```

Ruff (0.15.13)

[error] Probable insecure usage of temporary file or directory: "/tmp"

(S108)